components/board/jetson/data.go

```diff
@@ -17,15 +17,17 @@ const (
 	jetsonOrinNano = "jetson_orin_nano"
 )
 
+//TODO: fix ngpio numbers in pin definitions for the jetsonTX1, jetsonNano
+
 var claraAGXXavierPins = []genericlinux.PinDefinition{
-	{map[int]int{224: 134, 169: 106}, map[int]string{169: "PQ.06"}, "2200000.gpio", 7, 4, "MCLK05", "SOC_GPIO42", "", -1},
```

Comment:
I was going to have a question about how you decided which values to keep and which to remove, but on further thought, supposedly both of them used to work, so either one can be kept and it should be fine. The one kept might not correspond to the `"2200000.gpio"` further down the line, but the plan is to remove those, too, in the near future.